examples/00-mapdl-examples/3d_notch.py

```diff
@@ -66,19 +67,45 @@
 
 # mapdl.aplot(vtk=True, show_line_numbering=True)
 mapdl.lsla("S")
-mapdl.lplot(vtk=True, show_keypoint_numbering=True)
+
+pv.set_jupyter_backend("pythreejs")
+plotter = pv.Plotter(shape=(1, 3))
+
+plotter.subplot(0, 0)
+mapdl.lplot(
+    vtk=True, show_keypoint_numbering=True, plotter=plotter, return_plotter=True
+)
+plotter.add_title("Line Plot", font_size=12)
 mapdl.lsel("all")
 
 # plot the area using vtk/pyvista
-mapdl.aplot(vtk=True, show_area_numbering=True, show_lines=True, cpos="xy")
+plotter.subplot(0, 1)
+mapdl.aplot(
+    vtk=True,
+    show_area_numbering=True,
+    show_lines=True,
+    cpos="xy",
+    plotter=plotter,
+    return_plotter=True,
```

Comment:
``return_plotter`` argument is "hack" to avoid plotting before having already go through all the 3 subplots.

Hence, ideally we would like to not need that argument.